src/routes/auth.ts

```diff
@@ -43,7 +51,7 @@ router.get(
 
 router.get(
   "/auth/google/callback",
-  passport.authenticate("google", { failureRedirect: "/login" }),
+  passport.authenticate("google"),
```

Comment:
You missed me. I mean that we can use two cases:
- failureRedirect
- successRedirect